demo/api/src/products/generated/dto/product.sort.ts

```diff
@@ -6,7 +6,6 @@ import { IsEnum } from "class-validator";
 
 export enum ProductSortField {
     title = "title",
-    visible = "visible",
```

Comment:
Do we need to support sorting by status?

Reply:
you are right, inside one view can be multiple stati

changed, filter is now added